x-pack/plugins/secops/public/components/timeline/body/renderers/__snapshots__/plain_column_renderer.test.tsx.snap

```diff
@@ -17,7 +17,7 @@ exports[`plain_column_renderer rendering renders correctly against snapshot 1`]
         },
       }
     }
-    key="timeline-draggable-column-event.category-for-event-1"
+    key="timeline-draggable-column-event.category-for-event-1-event.category--Access"
```

Comment:
👍 

This is a helpful snapshot still and let me see that indeed the `id` as well as `key` are both unique which is a big plus.